src/Stratis.Bitcoin.Features.PoA/SlotsManager.cs

```diff
@@ -77,6 +76,7 @@ public uint GetMiningTimestamp(uint currentTime)
             return nextTimestampForMining;
         }
 
+        /// <summary>Determines whether timestamp is divisible by <see cref="PoANetwork.TargetSpacingSeconds"/>.</summary>
```

Comment:
` is divisible by <see cref="PoANetwork.TargetSpacingSeconds"/>` is an implementation detail.
"Determines whether timestamp is valid according to the network rules." is better.